hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/Alert.java

```diff
@@ -316,9 +322,103 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        return "Alert [alertId=" + alertId + ", status=" + status + ", ackTime=" + ackTime
-                + ", ackBy=" + ackBy + ", resolvedTime=" + resolvedTime + ", resolvedBy=" + resolvedBy + ", context="
-                + context + "]";
+        return "Alert{" +
+                "tenantId='" + tenantId + '\'' +
+                ", alertId='" + alertId + '\'' +
+                ", triggerId='" + triggerId + '\'' +
+                ", ctime=" + ctime +
+                ", evalSets=" + evalSets +
+                ", severity=" + severity +
+                ", status=" + status +
+                ", ackTime=" + ackTime +
+                ", ackBy='" + ackBy + '\'' +
+                ", resolvedTime=" + resolvedTime +
+                ", resolvedBy='" + resolvedBy + '\'' +
+                ", notes=" + notes +
+                ", trigger=" + trigger +
+                ", dampening=" + dampening +
+                ", resolvedEvalSets=" + resolvedEvalSets +
+                ", context=" + context +
+                '}';
+    }
```

Comment:
The previous toString() specifically left out evalSets and resolvedEvalSets because it's so verbose. (basically, make this 'thin').    Also, this is a definite formatting/generation discrepancy between IJ and Eclipse, not much we an do about it... But I do wish we could at least use [] as opposed to {} for consistency.